scripts/biomedical/diseaseOntology/README.md

```diff
@@ -0,0 +1,50 @@
+# Importing the Disease Ontology (DO) data
+
+## Table of Contents
+
+- [Importing the Disease Ontology (DO) data](#importing-the-disease-ontology-do-data)
+  - [Table of Contents](#table-of-contents)
+  - [About the Dataset](#about-the-dataset)
+  - [About the Import](#about-the-import)
+
+## About the Dataset
+
+### Download URL
+
+The human disease ontology data can be downloaded from their official github repository [here](https://www.vmh.life/#human/all). The data is in `.owl` format and had to be parsed into a `.csv` format (see [Notes and Caveats](#notes-and-caveats) for additional information on formatting).
+
+### Overview
+
+The Disease Ontology database provides a standardized ontology for human diseases, for the purposes of consistency and reusability. It has contains extensive cross mapping of DO terms to other databases, namely, MeSH, ICD, NCI’s thesaurus, SNOMED and OMIM. More information on the database can be found [here](https://disease-ontology.org).
+
+This directory stores the script used to convert the dataset obtained from DO into a modified version, for effective ingestion of data into the Data Commons knowledge graph.
+
+### Schema Overview
+
+The schema representing reaction, metabolite and microbiome data from VMH is defined in [DO.mcf](https://raw.githubusercontent.com/suhana13/ISB-project/main/combined_list.mcf) and [DO.mcf](https://raw.githubusercontent.com/suhana13/ISB-project/main/combined_list_enum.mcf).
```

Comment:
schema will be stored in ChemicalComounds.mcf and ChemicalCompoundsEnum.mcf

Reply:
@spiekos , should it be chemical_compounds.mcf or disease.mcf?